apps/website/astro.config.ts

```diff
@@ -22,6 +22,7 @@ export default defineConfig({
     rehypePlugins: [rehypeSlug, [rehypeAutolinkHeadings, { behavior: 'wrap' }], rehypeToc],
   },
   server: { port: 1700 },
+  scopedStyleStrategy: 'where',
```

Comment:
https://docs.astro.build/en/guides/upgrade-to/v3/#changed-default-scopedstylestrategy